floss/main.py

```diff
@@ -270,18 +271,16 @@ def set_color_config(color):
     colorama.just_fix_windows_console()
 
     if color == "always":
-        colorama.init(strip=False)
```

Comment:
lets remove the colorama import if its no longer used here. also the dependency in setup.py